src/index.html

```diff
@@ -11,6 +11,49 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      data-qa="big"
+      method="GET"
+      >
+      <label
+        for="largeInput"
+        class="large__bar"
+        >
+        <img
+          src="images/Search.svg"
+          alt="icon"
+          class="large__icon">
+        <input
+          class="input__large"
+          data-qa="keypress"
+          type="text"
+          id="largeInput"
+          placeholder="Try &quot;Los Angeles&quot;"
+          >
```

Comment:
```suggestion
        <img
          src="images/Search.svg"
          alt="icon"
          class="large__icon"
        >
        <input
          class="input__large"
          data-qa="keypress"
          type="text"
          id="largeInput"
          placeholder="Try &quot;Los Angeles&quot;"
        >
```
Closing tag should be under the opening. Please fix all cases
![image](https://user-images.githubusercontent.com/96409563/229042994-565b4076-0a3f-4695-938d-ab122ec84a8a.png)